src/lem/store.rs

```diff
@@ -753,6 +753,7 @@ impl<F: LurkField> Ptr<F> {
                     }
                     None => "<Malformed Comm>".into(),
                 },
+                Cproc => "<COPROCESSOR (TODO)>".into(),
             },
             Tag::Cont(_) => "<CONTINUATION (TODO)>".into(),
             _ => unreachable!(),
```

Comment:
do we have an issue open for the format TODOs?

Reply:
#711